### ISSUE TITLE
Error during installation

### ISSUE THREAD

**Issue by Traxko** [keybase/client#9087]
Hello, I tried installing both in a Windows administrator account and in a standard user account, and I received the same error message either way.  I've attached the log with the details.  It would be great if you can solve this installation bug.  Also, have you considered switching to UWP in Windows 10?  Microsoft recommends that all NEW apps be developed using UWP, not the older API's.

[Keybase_20170920143717.log](https://github.com/keybase/client/files/1389484/Keybase_20170920143717.log)

**Comment by maxtaco** [comment]
@zanderz can you investigate?

**Comment by Traxko** [comment]
I see you're using WiX to make the Keybase installer.  I blame this bug indirectly on WiX.  I wouldn't be surprised if you've already suffered a few headaches from using WiX.  If you switch to the modern "Universal Windows Platform" (UWP), there are multiple benefits, including much easier and better installation -- easier for both users and developers.  That's only one of MANY advantages of UWP.

**Comment by zanderz** [comment]
This failure is when you click on "view folders in Explorer", right? We've made improvements that will ship this week - apologies for the inconvenience.

**Comment by zanderz** [comment]
BTW, UWP apps do not work on Windows 7, 8, or 8.1, which still account for some 54% of all desktops (https://www.netmarketshare.com/operating-system-market-share.aspx). Aside from an easier installation experience, what other advantages would UWP bring?

**Comment by Traxko** [comment]
> "This failure is when you click on "view folders in Explorer", right?"

No, the failure occurs when trying to install Keybase -- when running the Keybase installer.  I'm unable to install Keybase.

> "UWP apps do not work on Windows 7, 8, or 8.1, "

Are you sure?  I read this:  "Windows 10 SDK works best on the Windows 10 operating system. And **it is also supported on**: Windows 8.1, Windows 8, Windows 7, Windows Server 2012, Windows Server 2008 R2, but not all tools are supported on these operating systems." ([Link](http://www.c-sharpcorner.com/article/how-to-install-windows-10-sdk-and-windows-10-emulators/))

Despite what that webpage says, you could be correct and yes you do make a good point, but here's another good point:  Keybase is a new project, and should a **new** project be developed for the **future** or for the **past**?

> "Aside from an easier installation experience, what other advantages would UWP bring?"

Multiple other advantages!  I'd like to write more on this topic but I have a pressing task that I must work on today, so unfortunately I'll have to end my message here for today.  Please also take into consideration that it's actually _your_ responsibility (not mine) to research UWP and make a decision.  I'm happy to help where possible but unfortunately I have only limited time and energy available.

**Comment by zanderz** [comment]
To see why installation fails, we'd also need to see the files named `%tmp%\Keybase_2017*KeybasePrograms.log`.
